app/classifier/task-nav.jsx

```diff
@@ -165,7 +166,7 @@ class TaskNav extends React.Component {
             >
               Back
             </button>}
-          {(!nextTaskKey && this.props.workflow.configuration.hide_classification_summaries && this.props.project && !disableTalk) &&
+          {(!nextTaskKey && this.props.workflow.configuration.hide_classification_summaries && this.props.project && !disableTalk && (!isFeedbackActive(this.props.project) && isThereFeedback(this.props.subject, this.props.workflow))) &&
```

Comment:
This will return a different value than the check that's been added to the `completeClassification` method in the classifier.